src/data/faq_de.json

```diff
@@ -1285,6 +1285,22 @@
                         "id": "positive_test",
                         "indexed": true,
                         "accordion": [
+                            {
+                                "title": "TAN-Verfahren",
+                                "anchor": "tan_procedure",
+                                "active": false,
+                                "textblock": [
+                                    "<div class='container'><div class='row'><div class='col-sm'><p><b>Zweck</b></p><p>Wenn Sie auf das Sars- CoV-2-Virus getestet werden und das Ergebnis digital in der Corona-Warn-App (CWA) empfangen haben, können Sie bei einem positiven Befund in der CWA andere Nutzer*innen warnen und so helfen, ggf. Kontaktketten frühzeitig zu unterbrechen.</p><p>Wird nach dem Einscannen des QR-Codes vom Formular 10C/OEGD (oder ein äquivalentes Formular) das Ergebnis nicht in die CWA übertragen, können Sie den positiven Befund trotzdem in der App registrieren und damit andere warnen, sofern Sie eine TAN von der Verifikations-Hotline empfangen.</p></div><div class='col-sm'><a href='/assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.pdf' target='_blank'><img src='/assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.jpg' alt='Positives Testergebnis teilen Anleitung PDF'></a></div></div></div>",
+                                    "<b>Ablauf</b>",
+                                    "<ul><li>Sie erreichen die Hotline ausschließlich telefonisch.\nRufnummer: 0800-7540002 (in Deutschland) bzw. +49.30.49875402 (international)\nWann Sie die Hotline erreichen, prüfen Sie bitte hier:\n<a href='#international_phone_numbers' target='_blank'>FAQ: So erreichen Sie uns</a></li><li>Schritt 1: Beantworten Sie die Fragen des Hotline-Mitarbeiters.</li><li>Schritt 2: Sie erhalten einen Rückruf von diesem Mitarbeiter, um Ihnen die TAN mitzuteilen. Diese TAN ist eine Stunde lang gültig.</li><li>Schritt 3: Entfernen Sie, falls registriert, den Test ohne Ergebnis aus der Corona-Warn-App.</li><li>Schritt 4: Im Home Screen der CWA wählen Sie die Schaltfläche „Weiter“ in der Funktion „Sie lassen sich testen?“\nUnter der Frage „Ihr PCR-Test war positiv?“ finden Sie die Funktion „TAN für PCR-Test eingeben“, wählen Sie diese Funktion und geben Sie die 10-stellige TAN ein, die Ihnen mitgeteilt wurde.</li><li>Ihr positives Testergebnis ist nun in der App registriert. Sie können Andere warnen, indem Sie Ihre Diagnoseschlüssel teilen.</li></ul>",

```

Comment:
I suggest to leave out "indem Sie ihre Diagnoseschlüssel teilen.", so just write:
"Nun können Sie über die App andere warnen."

This keeps it simple and less technical.

Reply:
Thanks for your suggestion, we replaced it in https://github.com/corona-warn-app/cwa-website/pull/3060/commits/6e7fa9f5f930a04a27c114ddc808a95ce8e62e3f